src/apiCalls.js

```diff
@@ -30,7 +30,16 @@ const updateData = (path, action, id) => {
 };
 
 export const apiCalls = {
+
```

Comment:
nice reusable functions for API calls

Reply:
Great ! will be super helpful moving forward